### PR TITLE
feat: fee model updates + 1.4.1 stage upgrade

### PR DESCRIPTION
**Pull request by AntonD3**
## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.
- [ ] Spellcheck has been run via `zk spellcheck`.

### CODE REVIEW

File: core/lib/dal/sqlx-data.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (StanislavBreadless) [inline core/lib/dal/sqlx-data.json:1]: delete this file

File: etc/upgrades/1705556759-fee-model-and-1.4.1/stage2/crypto.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "verifier": {
+    "address": "0x3390051435eCB25a9610A1cF17d1BA0a228A0560",
+    "txHash": "0x2c3b8915e8ebc617cc13e462df334f4197d48b19b302538a519f55acbda285cb"
+  },
+  "keys": {
+    "recursionNodeLevelVkHash": "0x5a3ef282b21e12fe1f4438e5bb158fc5060b160559c5158c6389d62d9fe3d080",
+    "recursionLeafLevelVkHash": "0x062362cb3eaf1f631406cbe19bf2a2c5d0d9ea69d069309a6003addae9f387be",
+    "recursionCircuitsSetVksHash": "0xf12256f7ba1ab223ba82abb19c8366c6677596897f9e5a82016977113e52ed5d"
```
Comment (perekopskiy) [inline etc/upgrades/1705556759-fee-model-and-1.4.1/stage2/crypto.json:9]: `recursionCircuitsSetVksHash` should be zero. Please also update tool to set it to zero always

Comment (AntonD3) [reply etc/upgrades/1705556759-fee-model-and-1.4.1/stage2/crypto.json:9]: https://github.com/matter-labs/zksync-era/blob/sb-short-term-fee-model-1-4-1/etc/env/base/contracts.toml#L43 - shouldn't it be taken from here? Or should we update it as well?

Comment (perekopskiy) [reply etc/upgrades/1705556759-fee-model-and-1.4.1/stage2/crypto.json:9]: No, it's but that it reads this variable

**Review by vladbochok — APPROVED** [review]

**Review by perekopskiy — DISMISSED** [review]

**Review by StanislavBreadless — DISMISSED** [review]

**Review by StanislavBreadless — DISMISSED** [review]

**Review by perekopskiy — DISMISSED** [review]

**Pull request merged**